### PR TITLE
[Merged by Bors] - perf (AlgebraicGeometry.Properties): Speed up one declaration 

### PR DESCRIPTION
**Pull request by mattrobball**
This speeds up on declaration by breaking up a defeq check into a single task.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mattrobball** [comment]
!bench

**Review by YaelDillies — APPROVED** [review]
Impressive!

maintainer merge

File: Mathlib/AlgebraicGeometry/Properties.lean [hunk 1]
```diff
@@ -94,6 +94,8 @@ instance {R : CommRingCat} [H : _root_.IsReduced R] : IsReduced (Scheme.Spec.obj
   intro x; dsimp
   have : _root_.IsReduced (CommRingCat.of <| Localization.AtPrime (PrimeSpectrum.asIdeal x)) := by
     dsimp; infer_instance
+  have : (Scheme.Spec.obj <| op R).presheaf = (Spec.structureSheaf R).presheaf := rfl
+  rw [this]
```
Comment (alreadydone) [inline Mathlib/AlgebraicGeometry/Properties.lean:98]:
```suggestion
  rw [show (Scheme.Spec.obj <| op R).presheaf = (Spec.structureSheaf R).presheaf from rfl]
```

**Review by jcommelin — APPROVED** [review]
Thanks :tada: 

If CI passes, please remove the label `awaiting-CI` and merge this yourself, by adding a comment `bors r+`.

bors d+

**Comment by mattrobball** [comment]
!bench

**Comment by mattrobball** [comment]
bors merge

**Pull request closed without merging**